client/src/components/scores/score.js

```diff
@@ -3,55 +3,55 @@
 export default {
 	template: `
 	<div class="card-section" ng-class="{ loading: loading }">
-	    <h4>
-	        <a editable-select="score.data.teamNumber" buttons="no" blur="submit" onaftersave="score.save()" e-ng-options="team.number as team.displayText for team in score.teams">
-	            <div ng-class="{'card-section alert' : score.teamNumberError()}">{{ score.teamText() }}</div>
-	        </a>
-	    </h4>
-	    <h6 class="subheader">
-	        <a editable-text="score.data.round" buttons="no" blur="submit" onaftersave="score.save()">
-	            <div ng-class="{'card-section alert' : score.roundError()}">{{ score.data.round || 'Missing round!'}}</div>
-	        </a>
-	    </h6>
-	</div>
-	<div class="stat text-center">
-	    <a editable-number="score.data.score" buttons="no" blur="submit" onaftersave="score.save()">
-	        {{ score.data.score || 0 }}
-	    </a>
-	</div>
-	<div class="card-divider">
-	    <a editable-text="score.data.referee" buttons="no" blur="submit" onaftersave="score.save()">
-	        {{ score.data.referee || 'No one' }}
-	    </a>
-	    &#160;on table&#160;
-	    <a editable-select="score.data.tableId" buttons="no" blur="submit" onaftersave="score.save()" e-ng-options="table.tableId as table.tableName for table in score.tables">
-	        {{ score.tableText() }}.
-	    </a>
-	</div>
-	<div class="card-section flex-child-shrink">

```

Comment:
Please bring back the class `flex-child-shrink` which was removed here in this commit